### ISSUE TITLE
Unable to get fully working

### ISSUE THREAD

**Issue by hakarune** [latera/homs#508]
I used the docker-compose instructions and got HOMS open on 3000 but there was no way to add processes and all the other advertised things on here; Camuda was working but there isn't a way to create a new process. I tried to get Activiti installed even and I couldn't access it to do anything. Is there something I'm missing?

**Comment by TimAle** [comment]
Hey, just to be sure, do you use this https://github.com/latera/homs#in-production manual?

> there isn't a way to create a new process.

Could you please clarify what exactly you want to do:
- to start a process from HOMS interface?
- to add a new order from HOMS interface?
- to add a new process definition to Camunda?

Maybe you can provide some screenshots or smth.

Activiti is definitely not supported anymore.

**Comment by hakarune** [comment]
Yes I followed that. I was able to create an order yaml file and upload it. But I could not for the life of me find out how to setup a process to do anything with the orders. I spent about 3 hours today trying to see if there were firewall issues, I wiped and redownloaded and retried, tried editing the .env file and nothing seemed to help. 

I was able to get on HOMS and create an order and do the yaml file stuff for custom orders setup, but Camunda had no way to create anything, but I could view the processes already on it. I was able to access the 9000 port program but it doesn't seem to do anything that I could see. I couldn't find anything that had a flow setup like the pictures I saw of this and it looks amazing.

**Comment by TimAle** [comment]
Well, it looks like you are trying to add new order type, right? (I can assume it because you uploaded order type YAML config to HOMS, but you didn't provide any information about what exactly you want to achieve).
Or you are talking about one of the demo processes? 
Could you describe your goal and provide a step-by-step description of your actions to achieve that?

**Comment by hakarune** [comment]
I set up just a test yaml to create a new order type to test out how it all works. When I tried to create a process that takes the order and processes it, I couldn't get anywhere. I tried finding a way in Camunda to create something since it had 3 demos already in it, but couldn't find a way. I tried to get this to work: http://hydra-oms.com/docs/system_workflow/#order_processing but I couldn't find a way to make the tasks appear or anything. I tried following this: http://hydra-oms.com/docs/setup/ & this: http://hydra-oms.com/docs/administration/ but couldn't get anything to work right. 

The goal was to have it setup to take a simple order (Beef) and then select numbers/weight of 5 sub-sections and then spit out a total price and a small invoice to a printer or as a PDF. 

I got the order creation set up and could input a new order for a customer, but then wasn't able to figure out how to get anything to work after that. I tried all the ports from the Docker-compose yaml, but only HOMS, Camunda and the thing on 9000 would connect.

**Comment by TimAle** [comment]
Sorry for the delay, now it's all clear :)

You should build your own process with the same process definition key as you created in HOMS and then deploy it to Camunda. 
For process building, you can use [Camunda Modeler](https://camunda.com/products/camunda-bpm/modeler/) and then you will be able to deploy your new process [directly from Modeler](https://docs.camunda.org/get-started/quick-start/deploy/#use-the-camunda-modeler-to-deploy-the-process). Another way is to pack a new process as `.war` file and put it in the `/camunda/webapps/` folder.
